makefile

```diff
@@ -1,7 +1,9 @@
 # Set to true to enable experimental multithreading
-THREADS = false
+THREADS ?= false
```

Comment:
This should not be set if absent, this is really experimental and currently full of bugs.